### ISSUE TITLE
Plan how to add `allowed_hosts` to all GA and Beta connectors

### ISSUE THREAD

**Issue by evantahler** [airbytehq/airbyte#21185]
1. Make a list of all GA and Beta connectors
2.  Create issues for all connectors and place them under this epic with a new tag, `allowed_hosts` so we can find them again
3. Divvy up the work somehow (between the entire connectors org + GL? just connector-ops on a focus day?) so we can get this done in a timely manner. 
OR
We write some automated tools to run a sync and capture a list of all the hosts we talk to, and then build the list pragmatically. 

This story is blocked by https://github.com/airbytehq/airbyte/issues/21183

This story will likely produce more stories to add allowed_hosts for connectors

**Comment by evantahler** [comment]
cc @YowanR this is where we will need the most help from GL.

**Comment by evantahler** [comment]
Ideas:
* Run a sync locally and look at network traffic (charles proxy?)
* read API docs?
* docker network tracing?

**Comment by evantahler** [comment]
cc @alafanechere 

As it's going to be a while (1 month+) until the platform is ready with their implementation, I think it might be OK to employ our new PR check (https://github.com/airbytehq/airbyte/pull/21684) and let the normal course of development take place and add the allowed hosts.  

Once the platform is ready, we can revisit this and make a list of the connectors which weren't organically caught in that time.
